### PR TITLE
CEDS-1524 draft dec pagination

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/config/AppConfig.scala [hunk 1]
```diff
@@ -105,6 +105,8 @@ class AppConfig @Inject()(
   lazy val draftTimeToLive: FiniteDuration =
     servicesConfig.getDuration("draft.timeToLive").asInstanceOf[FiniteDuration]
 
+  lazy val draftItemPerPage: Int = servicesConfig.getInt("draft.itemsPerPage")
```
Comment (ed0906) [inline app/config/AppConfig.scala]: I imagine the 50 limit will be service wide rather than just for drafts

Comment (ed0906) [reply app/config/AppConfig.scala]: But no harm keeping it as it is :)

Comment (SteveSugden) [reply app/config/AppConfig.scala]: You're probably right.  I'll rename it.

File: app/views/components/PaginationUtil.scala [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    if (max+1 <= pageCount && min-1 >= 1) {
+      (min-1) +: list :+ (max+1)
+    } else if (max+1 <= pageCount) {
+      list :+ (max+1)
+    } else if (min-1 >= 1) {
+      (min-1) +: list
+    } else {
+      list
+    }
+  }
```
Comment (ed0906) [inline app/views/components/PaginationUtil.scala]: What a fantastically well written, readable method 💯 I wonder who wrote it...

Comment (SteveSugden) [reply app/views/components/PaginationUtil.scala]: I did say it was ripped off in the PR request.   Take a bow Mr W

File: app/views/components/PaginationUtil.scala [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+ * limitations under the License.
+ */
+
+package views.components
+
+import scala.collection.immutable.Seq
+
+object PaginationUtil {
+
+  def surroundingPages(pageIndex: Int, pageCount: Int, available: Int): Seq[Int] = {
```
Comment (PatrykRudnicki) [inline app/views/components/PaginationUtil.scala]: Can we make this function without vars? just recursive function

File: app/views/components/PaginationUtil.scala (hunk 2 above)
Comment (PatrykRudnicki) [reply app/views/components/PaginationUtil.scala]: Hmm, I think removing parenthesis and putting in line with if will be more readable
```
if (max+1 <= pageCount && min-1 >= 1) (min-1) +: list :+ (max+1)
else if (max+1 <= pageCount) list :+ (max+1)
else if (min-1 >= 1) (min-1) +: list
else list
```

or just stay with current  format but without parenthesis. difficult to understand method's behaviour

File: app/views/components/PaginationUtil.scala (hunk 3 above)
Comment (SteveSugden) [reply app/views/components/PaginationUtil.scala]: This class was lifted from binding-tariffs.  I haven't really studied it to understand how it works, but it's tested/proven code and I'd rather just leave it as-is.  Unless we have nothing else to do.

**Pull request merged**